### DISCUSSION TITLE
Devo fixar a versão do SDK?

### DISCUSSION THREAD

**Discussion by victor-sc-92** [balta-io/forum#265] [Tira Dúvidas] [answered]
Aula: [.Net/Tipos de projeto](https://balta.io/player/assistir/5ef614da-366d-6156-d049-694800000000/0482ee81-8f83-4653-a421-000028010404)

Em alguns tutoriais, na hora de criarem o projeto, eles indicam fixar a versão do SDK.
Em projetos reais, isso realmente é feito? realmente tem essa necessidade? ou é algo mais de projetos antigos?
`$ dotnet new globaljson --sdk-version <versão>`

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @CodeAkio, como vai?

A versão do projeto é diferente deste item que você mencionou. Ele é válido para comandos do CLI.
Imagina que você tem o .NET 7 instalado, mas está trabalhando em um projeto com .NET 3.1, se houver algum item que o CLI do .NET tenha mudado do 3.1 para o 7, seu comando no terminal não irá rodar.

Neste caso, você pode utilizar uma versão específica do .NET SDK. Além disso podemos mudar a versão global do .NET também.
